ci/infra/meta.yml

```diff
@@ -0,0 +1,458 @@
+meta:
+  resources:
+    - name: ami-builder-configs
+      type: git
+      source:
+        branch: master
+        uri: https://github.com/dwp/ami-builder-configs.git
+        username: ((dataworks.enterprise_github_username))
+        password: ((dataworks-secrets.enterprise_github_pat))
+      webhook_token: ((dataworks.concourse_github_webhook_token))
+      check_every: 1h
+
+    - name: ami-builder-configs-pr
+      type: pull-request
+      source:
+        access_token: ((dataworks-secrets.concourse_github_pat))
+        repository: dwp/ami-builder-configs
+      webhook_token: ((dataworks.concourse_github_webhook_token))
+      check_every: 1m
```

Comment:
should be 720h and we should use the webhooks

Reply:
Done